h2o-r/tests/testdir_algos/grid/runit_grid_manual_export_resume.R

```diff
@@ -0,0 +1,43 @@
+setwd(normalizePath(dirname(R.utils::commandArgs(asValues=TRUE)$"f")))
+source("../../../scripts/h2o-r-test-setup.R")
+
+
+
+test.grid.resume <- function() {
+  iris.hex <- h2o.uploadFile(locate("smalldata/iris/iris.csv"), destination_frame="iris.hex")
```

Comment:
please use importFile instead

Reply:
Two tests affected. Replaced. Thank you.